.github/workflows/update_drupal_fingerprint.yml

```
@@ -0,0 +1,102 @@
+name: "Update Drupal Fingerprint"
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Run every Monday at 10:00 AM Romania time (8:00 AM UTC)
+    - cron: "0 8 * * 1"
+
+jobs:
+  update_drupal_fingerprint:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install requests
+
+      - name: Run create_hash_file.sh
+        run: |
+          chmod +x ./create_hash_file.sh
+          ./create_hash_file.sh
+
+      - name: Check for changes
+        id: git-check
+        run: |
+          if ! git diff --exit-code --quiet dscan/plugins/drupal/versions.xml; then
+            echo "changes=true" >> $GITHUB_OUTPUT
+            echo "Changes detected in dscan/plugins/drupal/versions.xml"
+          else
+            echo "changes=false" >> $GITHUB_OUTPUT
+            echo "No changes detected in dscan/plugins/drupal/versions.xml"
```

I would make this message a little bit more verbose:
```suggestion
            echo "No changes detected in dscan/plugins/drupal/versions.xml. Will update FORCE_DRUPAL_BUILD to false and skip creating a PR"
```